linux/Dockerfile.linux

```diff
@@ -0,0 +1,24 @@
+FROM alpine:edge
+
+# TODO: change to container registry
+COPY --from=clangbuiltlinux/containers:llvm-project llvm-project/llvm/build/bin /usr/local/bin
+COPY --from=clangbuiltlinux/containers:llvm-project /usr/local/lib /usr/local/lib
+COPY --from=clangbuiltlinux/containers:llvm-project /usr/local/include /usr/local/include
+RUN ln -s /usr/local/lib/x86_64-alpine-linux-musl/libc++abi.so.1 /usr/lib/. && \
+  ln -s /usr/local/lib/x86_64-alpine-linux-musl/libc++.a /usr/lib/. && \
+  ln -s /usr/local/lib/x86_64-alpine-linux-musl/libc++abi.a /usr/lib/. && \
+  ln -s /usr/local/lib/x86_64-alpine-linux-musl/libc++.so.1 /usr/lib/. && \
+  ln -s /usr/local/lib/x86_64-alpine-linux-musl/libunwind.so.1 /usr/lib/. && \
+  ln -s /usr/local/lib/x86_64-alpine-linux-musl/libunwind.a /usr/lib/.
+
+RUN apk add wget
+
+RUN wget --no-clobber --no-verbose https://git.kernel.org/torvalds/t/linux-5.18-rc6.tar.gz && \
+  tar xvf linux-5.18-rc6.tar.gz
+
+# rsync?
```

Comment:
`rsync` is what installs the headers: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Makefile#n1263

Reply:
haha, I always assumed it was for minimizing network traffic, but now that I think about it, pretty sure I know folks who use it for android over adb for first party platform development. Makes sense.